_layouts/landing.html

```diff
@@ -119,24 +119,41 @@ <h5 class="title">Reference</h5>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
-            <div class="col-xs-12 col-md-6"><a href="/get-started/swarm-deploy/">Scale apps using Swarm</a></div>
+            <ul class="nav nav-tabs">
+              <li class="active"><a data-toggle="tab" data-target="#includes">Explore Docker</a></li>
+              <li><a data-toggle="tab" data-target="#features">Get Support</a></li>
+              <li><a data-toggle="tab" data-target="#faqs">Browse FAQs</a></li>
+              </ul>
+              <div class="tab-content">
+              <div id="includes" class="tab-pane fade in active" markdown="1">
+               <div class="col-xs-12 col-md-6"><a href="/engine/reference/builder/">Write a Dockerfile</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/network/">Manage container networking</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/compose/compose-file/">Write a Docker Compose file</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/storage/">Work with volumes and bind mounts</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/04_sharing_app/">Share my image on Docker Hub</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/daemon/">Configure the Docker daemon</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/get-started/02_our_app/">Build and run an image</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/config/labels-custom-metadata/">Manage Docker objects</a></div>
+               <div class="col-xs-12 col-md-6"><a href="/build/building/multi-stage/">Multi-stage builds</a></div>
+              </div>
+              <div id="features" class="tab-pane fade" markdown="1">
+              <div class="col-xs-12 col-md-6"><a href="https://github.com/docker/for-win">Docker Desktop for Windows Github repo</a></div>
+              <div class="col-xs-12 col-md-6"><a href="/desktop/troubleshoot/overview/">Troubleshoot and diagnose</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://github.com/docker/for-linux">Docker Desktop for Linux Github repo</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://www.docker.com/company/contact/">Contact Docker</a></div>
+              <div class="col-xs-12 col-md-6"><a href="https://github.com/docker/for-mac">Docker Desktop for Mac Github repo</a></div></div>
+              <div id="faqs" class="tab-pane fade" markdown="1">
+              <div class="col-xs-12 col-md-6"><a href="https://docs.docker.com/desktop/faqs/windowsfaqs/#can-i-install-docker-desktop-on-windows-10-home">Can I install Docker Desktop on Windows 10 Home?</a></div>
```

Comment:
Instead of specific questions, could you add pointers to FAQ pages for Desktop, Compose, onboarding, SSO, pricing, etc?

Reply:
Sure, updated in new commit.